zrml/court/src/lib.rs

```diff
@@ -100,6 +105,33 @@ mod pallet {
 
     #[pallet::call]
     impl<T: Config> Pallet<T> {
+        #[pallet::weight(1_000_000_000_000)]
```

Comment:
I see, I previously thought that the `appeal` extrinsic is too big https://github.com/zeitgeistpm/zeitgeist/blob/3b789c14236acfd85b9f2e58425d50d5a0146f43/zrml/court/src/weights.rs#L133-L153 

But you mean the `#[pallet::weight(1_000_000_000_000)]` weight for `appeal` for the first court system. Since we are not using the first court system (MVP), as you mentioned, we can leave it as it is and rather benchmark the new court system.